test/performance/perf_test.lua

```diff
@@ -640,6 +699,29 @@ local cases = {
         row_name = row_name.insert,
     },
 
+    crud_batch_insert = {
+        call = 'crud.batch_insert',
+        params = batch_insert_params,
```

Comment:
But plain crud_insert should use multiple fiber parameters and report in CI/make coverage will be not the same as in this PR description.

Reply:
Comparison performed for `crud_insert_loop` and `crud_insert_many_different_batch_size`:

```lua
crud_insert_loop = {
        prepare = function(g)
            g.router:eval([[
                _insert_loop = function(space_name, tuples)
                    local results
                    local errors

                    for _, tuple in ipairs(tuples) do
                         local res, err = crud.insert(space_name, tuple)
                         if res ~= nil then
                             results = results or {}
                             table.insert(results, res)
                         end

                         if err ~= nil then
                             errors = errors or {}
                             table.insert(errors, err)
                         end
                    end

                    return results, errors
                end
            ]])
        end,
        call = '_insert_loop',
        params = batch_insert_params,
        matrix = batch_insert_cases.matrix,
        integration_params = integration_params,
        perf_params = batch_insert_comparison_perf,
        row_name = row_name.insert,
    },

    crud_insert_many_different_batch_size = {
        call = 'crud.insert_many',
        params = batch_insert_params,
        matrix = batch_insert_cases.matrix,
        integration_params = integration_params,
        perf_params = batch_insert_comparison_perf,
        row_name = row_name.insert_many,
    },
```

This test cases run with one fiber.

```lua
local batch_insert_comparison_perf = {
    timeout = 30,
    fiber_count = 1,
    connection_count = 1,
}
```